src/mcp_agent/tracing/tracer.py

```
@@ -0,0 +1,142 @@
+            AnthropicInstrumentor().instrument()
+        except ModuleNotFoundError:
+            logger.error(
+                "Anthropic otel instrumentation not available. Please install opentelemetry-instrumentation-anthropic."
+            )
+        try:
+            from opentelemetry.instrumentation.openai import OpenAIInstrumentor
+
+            OpenAIInstrumentor().instrument()
+        except ModuleNotFoundError:
+            logger.error(
+                "OpenAI otel instrumentation not available. Please install opentelemetry-instrumentation-anthropic."
```

Pylint (3.3.7)

[convention] Line too long (116/100)

(C0301)